pincer/commands/commands.py

```diff
@@ -802,16 +965,65 @@ async def initialize(self):
             return
 
         ChatCommandHandler.has_been_initialized = True
+
+        self.__build_local_commands()
         await self.__get_existing_commands()
         await self.__remove_unused_commands()
         await self.__add_commands()
 
 
-def hash_app_command(command: AppCommand) -> int:
-    return hash_app_command_params(command.name, command.guild_id, command.type)
+def hash_app_command(
```

Comment:
since we can use the `__hash__` dunder, shouldn't this function made pythonic "private"?

Reply:
I think it can be protected. Im pretty sure imports wouldn't work if it was private.